ako-gateway-api/lib/lib.go

```diff
@@ -31,10 +31,14 @@ func InformersToRegister(kclient *kubernetes.Clientset) ([]string, error) {
 	// Services, Endpoints, Secrets, ConfigMaps.
 	allInformers := []string{
 		utils.ServiceInformer,
-		utils.EndpointInformer,
 		utils.SecretInformer,
 		utils.ConfigMapInformer,
 	}
+	if lib.AKOControlConfig().GetEndpointSlicesEnabled() {
+		allInformers = append(allInformers, utils.EndpointSlicesInformer)
+	} else if lib.GetServiceType() != lib.NodePortLocal {
```

Comment:
This might be conflicted after https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/1466/files#diff-4d3654efe8d8ca4cd41756b7b11781c301e4521e1d6431736dfefcaae0999e7a this Pr is merged

Reply:
will be taken up when resolving conflicts